applications/api-cat/src/main/java/no/acat/harvester/ApiHarvester.java

```diff
@@ -73,16 +73,22 @@ public void harvestAll() {
     }
 
     List<ApiRegistrationPublic> getApiRegistrations() {
+
+        List<ApiRegistrationPublic> result = new ArrayList<>();
+
         List<ApiRegistrationPublic> apiRegistrationsFromCsv = getApiRegistrationsFromCsv();
-        logger.info("Loaded registrations from csv {}", apiRegistrationsFromCsv.size());
+        if (apiRegistrationsFromCsv != null) {
+            logger.info("Loaded registrations from csv {}", apiRegistrationsFromCsv.size());
```

Comment:
Småpirk for lesbarhet: om du skriver ut antallet apiRegistrations eller "Loaded" er det lettere å lese og forstå både i kode og i output.